lib/mayaUsd/fileio/primUpdater.h

```diff
@@ -115,6 +118,9 @@ class UsdMayaPrimUpdater
     MAYAUSD_CORE_PUBLIC
     UsdPrim getUsdPrim() const;
 
+    MAYAUSD_CORE_PUBLIC
+    const UsdMayaPrimUpdaterContext* getContext() const { return _context; }
```

Comment:
Is there a reason why it is returned by pointer? It seems to me it should always be non-null. All callers will need to check for nullity.

Reply:
Great point, could be a reference.  I don't think we should block the pull request on this, but I'll make a note for the follow-up.